app/controllers/tt/api/v1/visitor_details_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Tt::Api::V1::VisitorDetailsController < Tt::Api::V1::ApplicationController
+  def show
+    render json: TestTrack::FakeServer.visitor_details
```

Comment:
I noticed that we're doing more complicated stuff to model out this fake data for other models, when really it seems like we just want to dump out a pre-formed json response. am I thinking about that wrong? this seemed like the simplest way to achieve that.